Let's not expose the underlying Method.

shoumikhin opened the pull request:
Summary: .

Differential Revision: D80595270

JacobSzwejbka reviewed (approved):
Review automatically exported from Phabricator review in Meta.

shoumikhin committed:
Let's not expose the underlying Method. (#13543)

Summary:

.

Reviewed By: JacobSzwejbka, mergennachin

Differential Revision: D80595270

mergennachin reviewed (changes requested):
Okay we missed the 1.0 release to remove 

We'll have to comply with two releases now
